pkg/controller/migmigration/hooks.go

```diff
@@ -15,7 +15,8 @@ import (
 	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-const HookJobFailedLimit = 5
+const HookJobFailedLimit = 6
```

Comment:
Why is this supposed to be 6 rather than 5? Just curious.

Reply:
The default BackOffLimit (that we use) is 6, not 5. It will generate 6 failures before giving up.